metaci/testresults/robot_importer.py

```diff
@@ -294,23 +319,11 @@ def export_robot_test_results(flowtest) -> None:
         "build": {
             "name": flowtest.build_flow.build.plan.name,
             "number": flowtest.id,
-            "url": "http://www.example.com",  # build.get_external_url() returns 'detail': [{'loc': ['body', 'build', 'url'], 'msg': 'URL host invalid, top level domain required', 'type': 'value_error.url.host'}] when sent to gus-bus,
+            "url": "https://www.example.com/",  # flowtest.build_flow.build.get_external_url(),  # returns 'detail': [{'loc': ['body', 'build', 'url'], 'msg': 'URL host invalid, top level domain required', 'type': 'value_error.url.host'}] when sent to gus-bus,
```

Comment:
What do we need to do to fix this? I don't quite understand what the comment is telling me.

Reply:
@davisagli I need help with this.

Reply:
https://github.com/SFDO-Tooling/gus-bus/blob/1f1fdeb5bf9200bfacc1216746d9e9a53c653534/gus_bus/testmanager.py#L38 is validating that the `url` is a URL, and apparently Django is using a URL with a hostname that has no top-level domain and pydantic doesn't like it.

You can make the tests use a different hostname (example.com?) by setting SITE_URL in https://github.com/SFDO-Tooling/MetaCI/blob/main/config/settings/test.py